ops/jenkins/bfd-build-server-load-controller/build-push.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+# SCRIPT_DIR will return the directory where this script is located, and CONTEXT_DIR will be the
+# directory of the Docker build context (locust_tests). This way, this script can be called from
+# _any_ directory and there will be no issues
+SCRIPT_DIR="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+CONTEXT_DIR="${REPO_ROOT}/apps/utils/locust_tests/"
+
+# Constants
+GIT_SHORT_HASH="$(git rev-parse --short HEAD)"
+AWS_REGION="us-east-1"
+PRIVATE_REGISTRY_URI="$(aws ecr describe-registry --region "$AWS_REGION" | jq -r '.registryId').dkr.ecr.${AWS_REGION}.amazonaws.com"
+IMAGE_NAME_NODE="${PRIVATE_REGISTRY_URI}/bfd-mgmt-server-load-controller"
+SSM_IMAGE_TAG_NODE="/bfd/mgmt/server/nonsensitive/server_load_controller_latest_image_tag"
+
+# Overridable defaults
+DOCKER_TAG="${DOCKER_TAG_OVERRIDE:-"$GIT_SHORT_HASH"}"
+DOCKER_TAG_LATEST="${DOCKER_TAG_LATEST_OVERRIDE:-"latest"}"
+
+# Build tagged node image
+docker build "$CONTEXT_DIR" \
+  --file "$SCRIPT_DIR/Dockerfile" \
+  --tag "${IMAGE_NAME_NODE}:${DOCKER_TAG}" \
+  --tag "${IMAGE_NAME_NODE}:${DOCKER_TAG_LATEST}" \
+  --platform "linux/amd64"
+
+# Get registry password and tell docker to login
+aws ecr get-login-password --region "$AWS_REGION" | docker login --username AWS --password-stdin "$PRIVATE_REGISTRY_URI"
+
+# Push image to ECR
+docker push "$IMAGE_NAME_NODE" --all-tags
+
+# Put the image tag in SSM
+aws ssm put-parameter \
+  --name "$SSM_IMAGE_TAG_NODE" \
+  --value "${IMAGE_NAME_NODE}:${DOCKER_TAG}" \
+  --overwrite \
+  --type String \
+  --region "$AWS_REGION"
```

Comment:
Can we add our default tags to any parameters we are putting into SSM via CLI? This will help note what params are not in IaC and are created via CLI. Thanks!

Reply:
Sorry, I've read this comment a few times and I'm not sure what you mean?

Reply:
No worries. I was wondering if you could add our default tags to the put-parameter call on lines L36-L41. Something like 
```
aws ssm put-parameter \
  --name "$SSM_IMAGE_TAG_NODE" \
  --value "${IMAGE_NAME_NODE}:${DOCKER_TAG}" \
  --overwrite \
  --type String \
  --tags Key=Environment,Value=mgmt Key=stack,Value=mgmt Key=Terraform,Value=False \
  Key=application,Value=bfd Key=business,Value=oeda \
  --region "$AWS_REGION"
```

Reply:
Of course, whatever tags you think are appropriate would suffice here. Thanks!

Reply:
Ahh, I see. Good call

Reply:
I ended up doing this OOB since you cannot overwrite SSM params while also passing in tags, and writing code to handle this conditional each time the image is built seems extraneous to me...thoughts?